index.html

```diff
@@ -5,44 +5,129 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
 </head>
 <body>
    <header>
+       <div class="upper_navbar">
+           <ul>
+            <li><a href="#"><img src="images/linkedinlogo.png" alt="Linked in nav item"></a></li>
+            <li><a href="#"><img src="images/twitterlogo.png" alt="Linked in nav item"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>          
+         </ul>
+       </div>
        <nav>
-           <button type="button"><img src="images/hamburger.png" alt="Hamburger button"></button>
-       </nav>
+           <button type="button" id="hamburger-btn"><img src="images/hamburger.png" alt="Hamburger button"></button>
+           <div class="desktop-navbar">
+                <a class="logo_link" href="#"><img src="images/Screenshot from 2022-02-16 15-13-24.png" alt="Homifyers logo link"></a>
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#categories">Categories</a></li>
+                    <li><a href="#tstmons">Top Customers</a></li>
+                    <li><a href="#brands">Brands</a></li>
+                    <li><a href="#wrhs">Visit</a></li>
+                </ul>
+            </div>
+        </nav>
    </header>
    <main>
-       <sectiom class="hero">
+       <section class="hero" id="hero">
             <h3>Hello! Online Available Furniture</h3>
             <h1>Beautifully Designed Furniture</h1>
             <p class="msg">A website where all kind of furniture is available. Beatifully designed and best furniture is available and it's delivery is also available. 100%% best quality furniture is available at your door step.
                 You can choose and buy your favourite furniture from our website and we can deliver it to you.
             </p>
-            <h3 class=""></h3>
-            <p>@Furnishers Islamabad, Pakistan</p>
-       </sectiom>
-       <section class="works">
+            <h2 class="current-time"></h2>
+            <p class="tit">@Furnishers Islamabad, Pakistan</p>
+       </section>
+       <section class="works" id="categories">
             <h2>Categories</h2>
             <div class="ctsprtr"></div>
             <ul>
-                <li><div class="bed"><img src="images/bed.png" alt=""></div><h5>Beds</h5><p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Facilis omnis assumenda architecto maiores sunt quisquam ipsa eligendi natus voluptatibus earum ab, provident minus veniam labore, eius deserunt qui unde dolores?</p></li>
```

Comment:
- [ ] Kindly add a descriptive placeholder in all your `alt` attributes, this is best for accessibility